chia/data_layer/data_layer_wallet.py

```diff
@@ -64,18 +63,30 @@ class DataLayerWallet:
     interface used by datalayer for interacting with the chain
     """
 
+    @classmethod
+    async def create(
+        cls: Type[_T_DataLayerWallet],
+        wallet_state_manager: Any,
+        wallet: Wallet,
+        wallet_info: WalletInfo,
+        name: Optional[str] = None,
+    ) -> _T_DataLayerWallet:
+        self = cls()
```

Comment:
This should be a dataclass so it has a proper init that can be used instead of just a totally freeform class instance created on the fly.